script.js

```diff
@@ -157,14 +162,15 @@ function JSprojectDesktop() {
 
   function popupwindowDesktopFunc() {
     overlay.classList.add('active');
```

Comment:
- [ ] I kindly suggest that you implement the project modal the same way it is designed on the Figma. Find a video explanation below:
- [ ] I kindly suggest that only one project should be on display when the see project button is clicked. Currently when the see project button is clicked, multiple projects are displayed.

https://www.loom.com/share/c4f920d4b2eb46ada3ff054c607e0daa